docs/modules/ROOT/pages/tutorial-standalone.adoc

```diff
@@ -0,0 +1,371 @@
+= Stand-Alone Deployment
+:icons: font
+:imagesdir: ../assets/images
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+[abstract]
+While CMOS is designed as an all-in-one stack, it is also possible to deploy all its components separately.
+This tutorial will show you how to configure and deploy each component.
+
+include::partial$tutorial.adoc[]
+
+== Overview
+
+Couchbase Monitoring and Observability Stack (also known as CMOS) is a simple, out-of-the-box solution built using industry-standard tooling to observe the state of a running Couchbase cluster.
+
+If you already have all the components of the stack in your deployment, it is easy to connect them all into a complete monitoring solution and take advantage of all the features, including the advanced health checking capabilities of the Couchbase Cluster Monitor.
+
+In this tutorial, we will use Docker to configure all the components individually from scratch.
+However, if you already have some of these components deployed, you can skip the respective sections. 
+You may wish to read xref:integrating-with-existing-deployments.adoc[].
+
+== Pre-Requisites
+
+We assume that you already have a Couchbase Server cluster running.
+If not, refer to the xref:7.0@server:install:getting-started[installation options] to deploy a cluster.
+
+We also assume that you have Docker installed.
```

Comment:
Suggestion of where/how to install docker, if that is allowed as might be different company